experimental/InvariantTheory/src/InvariantTheory.jl

```diff
@@ -675,3 +720,78 @@ function affine_algebra(R::RedGroupInvarRing)
   end
   return domain(R.presentation), R.presentation
 end
+
+
+###################
+#In the case there is no reynolds operator
+###################
+
+#Algorithm 4.5.1 in Computational Invariant Theory by Derksen and Kemper  
```

Comment:
please add "second edition" or so (and/or reference `[DK15]` which is the key we use in our .bib file